ray-operator/test/e2e/support.go

```diff
@@ -213,4 +210,46 @@ func deployRedis(t Test, namespace string, password string) {
 		TestApplyOptions,
 	)
 	assert.NoError(t.T(), err)
+
+	checkDBSize := func() string {
+		req := t.Client().Core().CoreV1().RESTClient().Post().Resource("pods").
+			Namespace(namespace).Name("redis").SubResource("exec").
+			VersionedParams(&corev1.PodExecOptions{
+				Container: "redis",
+				Command:   dbSizeCmd,
+				Stdin:     false,
+				Stdout:    true,
+				Stderr:    true,
+			}, scheme.ParameterCodec)
+
+		stdout := bytes.NewBuffer(nil)
+		stderr := bytes.NewBuffer(nil)
+
+		config := t.Client().Config()
+
+		executor, err := remotecommand.NewSPDYExecutor(&config, "POST", req.URL())
+		if err != nil {
+			t.T().Fatalf("failed to create executor: %v", err)
+		}
+
+		err = executor.StreamWithContext(t.Ctx(), remotecommand.StreamOptions{
+			Stdout: stdout,
+			Stderr: stderr,
+		})
+		if err != nil {
+			t.T().Fatalf("failed to execute: %v", err)
+		}
+
+		return strings.TrimSpace(stdout.String() + stderr.String())
+	}
+	return func() {
```

Comment:
Maybe only return `checkDBSize` and use `Eventually` in `raycluster_gcsft_test.go`?

Reply:
👍